src/components/panels/Machine/ConfigFilesPanel.vue

```diff
@@ -512,7 +512,7 @@
             </panel>
         </v-dialog>
 
-        <v-snackbar v-model="uploadSnackbar.status" :timeout="-1" :value="true" fixed right bottom dark>
+        <v-snackbar v-model="uploadSnackbar.status" :timeout="-1" :value="true" fixed right bottom>
```

Comment:
I think this will be wrong, if you just delete the dark option with no condition.